### DISCUSSION TITLE
Bug report. Blacklist isn't working since last update.

### DISCUSSION THREAD

**Discussion by darklimiex** [aclap-dev/video-downloadhelper#183] [Help and Q&A]
### Your question

Don't know if was already reported, tried to search but didn't found the topic. Since last update ( 8.2.0.23) the blacklist function is not working on Firefox. 

Thanks for the amazing add-on.

### Addon & CoApp version and details

Firefox 122.0.1
Vdh 8.2.0.23
VdhCoApp 2.0.17

**Comment by paulrouget** [comment]
Thank you for the bug report!

It seems that we broke something indeed.

Actually, I'm curious, in which case you find blacklisting useful? It's a very underused feature we introduced very long time agao, and I've been wondering how people use it.

**Reply by DBLClick** [reply]
Hopefully This could be fixed and total domain detection would be implemented.
